addons/sale/views/sale_portal_templates.xml

```diff
@@ -131,71 +131,50 @@
                     <t t-set="title">
                         <h2 class="mb-0"><b t-field="sale_order.amount_total"/> </h2>
                     </t>
-
                     <t t-set="entries">
                         <ul class="list-group list-group-flush flex-wrap flex-row flex-lg-column">
                             <li class="list-group-item flex-grow-1">
-                                <a role="button" class="btn btn-block btn-secondary" target="_blank" t-att-href="sale_order.get_portal_url(report_type='pdf')"><i class="fa fa-print mr-1"/>Print</a>
-
-                                <t t-if="sale_order.state in ('draft', 'sent', 'waiting_date')">
-                                    <t t-set="to_accept" t-value="not sale_order.is_expired and not sale_order.signature and sale_order.has_to_be_signed()"/>
-                                    <t t-set="to_accept_and_pay" t-value="not sale_order.is_expired and sale_order.has_to_be_paid()"/>
-
-                                    <t t-if="to_accept or to_accept_and_pay or sale_order.is_expired">
-                                        <a role="button" t-if="to_accept" class="btn btn-primary btn-block" data-toggle="modal" data-target="#modalaccept" href="#">
-                                            <i class="fa fa-check"/> Accept &amp; Sign
-                                        </a>
-                                        <a role="button" t-elif="to_accept_and_pay" class="btn btn-success btn-block" data-toggle="modal" data-target="#modalaccept" href="#">
-                                            <i class="fa fa-check"/> <t t-if="not sale_order.signature">Accept &amp; </t>Pay
-                                        </a>
-                                        <t t-if="sale_order.is_expired">
-                                            <div class="alert alert-warning small mt-3 mb-1 py-1 px-2"><b class="d-block mb-1">This offer expired!</b>
-                                                <a role="button" href="#discussion" class="btn btn-info btn-block py-1 o_normalwhitespace"> Get a new quotation.</a>
-                                            </div>
-                                        </t>
-                                    </t>
-                                </t>
-
-                                <t t-if="need_payment or (sale_order.state in ('sale') and sale_order.has_to_be_paid() and tx_state != 'done')">
-                                    <a role="button" class="btn btn-success btn-block" data-toggle="modal" data-target="#modalaccept" href="#">
-                                        <i class="fa fa-check"></i> Pay Now
-                                    </a>
-                                </t>
+                                <a t-if="sale_order.has_to_be_signed(True)" role="button" t-attf-class="btn btn-{{ 'primary' if sale_order.has_to_be_paid() else 'success'}} btn-block mb8" data-toggle="modal" data-target="#modalaccept" href="#">
```

Comment:
Why using success in some cases ?
Success green color is ugly, only used in publish widget if I am not mistaken.
Using btn-primary looks way more clean imo
+ it will ease the condition